Modules/ThirdParty/MINC/src/libminc/volume_io/Volumes/input_nifti.c

```diff
@@ -10,6 +10,9 @@
 
 #include "nifti1.h"
 #include "nifti1_io.h"
+#ifdef DEBUG
```

Comment:
`DEBUG` or `_DEBUG` like the other file?

Reply:
There was some differences between the files, but I kept the pattern in upstream.